lib/mapping.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
   ircToSlack: function(originalMessage) {
     var modifiedMessage = originalMessage;
     _.each(userMap, function(val, key, list) {
-      var re = new RegExp("@?" + key, 'gi');
+      var re = new RegExp("@?" + key + '\s', 'gi');
```

Comment:
this would also break things like "rob: what do you mean?".  what if instead of a space after you looked for any non-letter character?